**Unit 4: Protein Engineering and Proteomics** <span id="4"></span> 
*A proteome is the collection of all proteins expressed by a genome, and
proteomics analyzes the proteome.  Proteins serve almost every purpose
imaginable, from catalyzing reactions to providing structural support. 
Research into proteomics can be more complex than genomics, as they are
inherently much more versatile biomolecules.  *  
 *             
 This unit will summarize the current state of proteomics and protein
engineering.  We will first look at how a protein is currently studied,
such as isolating and characterizing a protein.  We will then look at
engineering novel proteins based on current techniques and knowledge.*

**Unit 4 Time Advisory**  
This unit should take you approximately 18 hours to complete.  
  
 ☐    Subunit 4.1: 1.5 hours  
  
 ☐    Subunit 4.2: 2.5 hours  
  
 ☐    Subunit 4.3: 2.0 hours  
  
 ☐    Subunit 4.4: 2.5 hours  
  
 ☐    Subunit 4.5: 3.5 hours  
  
 ☐    Subunit 4.6: 6.0 hours  
  
☐    Subunit 4.6.1: 0.5 hour  
  
 ☐    Subunit 4.6.2: 1.0 hour  
  
 ☐    Subunit 4.6.3: 1.0 hour  
  
 ☐    Subunit 4.6.4: 0.5 hour  
  
 ☐    Subunit 4.6.5: 0.5 hour  
  
 ☐    Subunit 4.6.6: 0.5 hour  
  
 ☐    Subunit 4.6.7: 1.0 hour  
  
 ☐    Subunit 4.6.8: 1.0 hour

**Unit4 Learning Outcomes**  
Upon successful completion of this unit, the student will be able to:  
  
-   Describe how proteins are isolated, characterized, and sequenced.
-   Compare and contrast protein interaction techniques.
-   Discuss and contrast the design of recombinant proteins expression
    in prokaryotes and eukaryotes.
-   Identify challenges during the production of biologically active
    proteins.
-   Describe novel function-oriented protein design.

**4.1 Protein Visualization and Identification** <span id="4.1"></span> 
**4.1.1 SDS-PAGE (Sodium Dodecyl Sulfate-Polyacrylamide Gel
Electrophoresis)** <span id="4.1.1"></span> 
-   **Reading: Davidson College: Department of Biology’s “SDS-PAGE
    (PolyAcrylamide Gel Electrophoresis)”**
    Links: Davidson College: Department of Biology’s  [“SDS-PAGE
    (PolyAcrylamide Gel
    Electrophoresis)”](http://www.bio.davidson.edu/courses/genomics/method/sdspage/sdspage.html)
    (HTML)  
        
     Instructions: Please study this page.  Please note that
    electrophoresis can be used to separate substances, which have
    electrical charge.  In the SDS-PAGE technique, proteins are
    associated with SDS, thus they become negatively charged.
     Furthermore, the smaller SDS-associated proteins will travel faster
    in the electric field than the larger ones.     
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.1.2 Western Blot** <span id="4.1.2"></span> 
-   **Reading: Davidson College: Department of Biology’s “Western Blot
    Procedure”**
    Links: Davidson College: Department of Biology’s “[Western Blot
    Procedure](http://www.bio.davidson.edu/courses/genomics/method/Westernblot.html)”
    (HTML)  
        
     Instructions: Please study the outline of the Western blot
    analysis.  This procedure is also called immunoblot technique,
    because it is utilizing antibodies to detect specific proteins in a
    mixture.  In reality, antibodies are not absolutely specific, thus
    this method may produce a false positive or negative signal.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Lecture: Journal of Visualized Experiments: Gallagher and
    Chakavarti’ s “Immunoblot Analysis”**
    Link: *Journal of Visualized Experiments:  *Gallagher and
    Chakavarti’ s “[Immunoblot
    Analysis](http://www.jove.com/details.stp?ID=759)” (Adobe Flash)  
        
     Instructions: Please watch the video (16 minutes) for a
    step-by-step tutorial on the immunoblot technique.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Assessment: The University of Arizona: The Biology Project’s
    “Introduction to Western Blot Activity”**
    Link: The University of Arizona:  The Biology Project’s
    “[Introduction to Western Blot
    Activity](http://www.biology.arizona.edu/immunology/activities/western_blot/w_main.html)”
    (HTML)  
        
     Instructions:  In order to assess your knowledge, follow the
    instructions and complete the four problems at the end of this
    activity.  Answer key is provided.  You can progress page by page by
    clicking on the “Next” bottom toward the bottom of the page.  
        
     Note: One animation requires a QuickTime plugin.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.2 Protein Isolation and Purification** <span id="4.2"></span> 
**4.2.1 HPLC (High-Performance Liquid Chromatography)** <span
id="4.2.1"></span> 
-   **Reading: Jim Clark’s ChemGuide: “High-Performance Liquid
    Chromatography—HPLC”**
    Link: Jim Clark’s *ChemGuide:* “[High-Performance Liquid
    Chromatography—HPLC](http://www.chemguide.co.uk/analysis/chromatography/hplc.html)”
    (HTML)  
        
     Instructions: Please study this page.  Please note that HPLC is
    used to purify peptides; it is not a typical protein purification
    method.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.2.2 Affinity Chromatography** <span id="4.2.2"></span> 
-   **Reading: John W. Kimball's Biology Pages: “Affinity
    Chromatography”**
    Link: John W. Kimball's Biology Pages:  “[Affinity
    Chromatography](http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/A/AffinityChrom.html)”
    (HTML)  
        
     Instructions: Please study this page.  Please note that affinity
    chromatography is a very efficient purification technique as long as
    the immobilized molecule binds strongly and specifically to the
    protein of interest.  Antibody-antigen interaction is an example of
    strong and specific binding.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: Davidson College: Department of Biology’s “Affinity
    Chromatography Method”**
    Link: Davidson College: Department of Biology’s “[Affinity
    Chromatography
    Method](http://www.bio.davidson.edu/courses/genomics/method/Affinity.html)”
    (HTML)  
        
     Instructions: Please study the outline of the Affinity
    Chromatography Method.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.2.3 Ion Exchange Chromatography** <span id="4.2.3"></span> 
-   **Reading: Florida State University: Michael Blaber’s “Protein
    Purification: Column Chromatography—Ion Exchange; Dialysis and
    Concentration”**
    Link: Florida State University:  Michael Blaber’s “[Protein
    Purification: Column Chromatography—Ion Exchange; Dialysis and
    Concentration](http://www.mikeblaber.org/oldwine/bch5425/lect30/lect30.htm)”
    (HTML)  
        
     Instructions: Please study this page.  Please note that proteins
    are typically purified from a biological sample.  Ion exchange
    chromatography is a low efficiency, but inexpensive purification
    technique.  It is oftentimes employed to remove substances from a
    complex mixture, which would otherwise interfere with the following
    specific purification step.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.3 Protein Sequencing** <span id="4.3"></span> 
**4.3.1 Edman Degradation** <span id="4.3.1"></span> 
-   **Reading: Wikibooks’ “Protein Primary Structure”**
    Link: Wikibooks’ “[Protein Primary
    Structure](https://resources.saylor.org/archived/wp-content/uploads/2011/06/Protein-Primary-Structure.pdf)”
    (PDF)  
        
     Instructions: Please learn the “Edman Degradation” section on this
    page.  
        
     Terms of Use: The article above is released under a [Creative
    Commons Attribution-Share-Alike License
    3.0](http://creativecommons.org/licenses/by-sa/3.0/) (HTML).  You
    can find the original Wikibooks version of this article
    [here](http://en.wikibooks.org/wiki/Proteomics/Protein_Primary_Structure/Sequencing_Methods)
    (HTML).

-   **Lecture: YouTube: Lamechivanes’ “Sequence Determination One
    Terminal Unit at a Time”**
    Link: Video Lecture: YouTube: Lamechivanes’ “[Sequence Determination
    One Terminal Unit at a
    Time](http://www.youtube.com/watch?v=d0mZ0n_AHnI)” (YouTube)  
        
     Instructions: Please watch the video (3 minutes) for a step-by-step
    tutorial on Edman degradation.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.3.2 Mass Spectrometry** <span id="4.3.2"></span> 
-   **Reading: The University of Leeds: Alison Ashcroft’s “An
    Introduction to Mass Spectrometry”**
    Link: The University of Leeds:  Alison Ashcroft’s “[An Introduction
    to Mass
    Spectrometry](http://www.astbury.leeds.ac.uk/facil/MStut/mstutorial.htm)”
    (HTML)  
        
     Instructions: Please study sections “1. What Is Mass Spectrometry
    (MS)? What Information Does Mass Spectrometry Provide?” and the
    content of subsections “4.1 Introduction,” “8.1 Tandem Mass
    Spectrometry,” “8.2 Tandem Mass Spectrometry Analyses,” and “8.3
    Peptide Sequencing by Tandem Mass Spectrometry “ on this page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Assessment: The Saylor Foundation: "BIO403 Unit 4.3 Assessment"**
    Link: The Saylor Foundation: "[BIO403 Unit 4.3
    Assessment](http://www.saylor.org/content/bio403/assessments/BIO403_Unit_4_3_assessment_Mass_Spec.html)"
    (HTML)  
        
     Instructions: You will find link to the "Mass Spectrometry"
    assessment on this page. This is a multiple choice assessment with
    one correct answer. Clicking on an answer will bring you to another
    page. If your answer is correct, then it is acknowledged with a
    short explanation. Please read the explanation carefully. If you
    clicked on the wrong answer, then the click will bring you to a
    tutorial page. Please study the tutorial page carefully. You will be
    prompted to return to the assessment and complete it again. Please
    note that mass spectrometry is a widely accepted confirmatory
    technique for the identification of organic molecules.

**4.4 Protein Interaction Methods** <span id="4.4"></span> 
**4.4.1 Yeast Two-Hybrid System** <span id="4.4.1"></span> 
-   **Reading: John W. Kimball’s Biology Pages: “Proteomics”**
    Link: John W. Kimball’s Biology Pages:
     “[Proteomics](http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/P/Proteomics.html)”
    (HTML)  
        
     Instructions: Please study “The Yeast Two-Hybrid System” section on
    this page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the web page above.

**4.4.2 Co-Immunoprecipitation** <span id="4.4.2"></span> 
-   **Reading: National Center for Biotechnology Information’s
    Bookshelf: Garland Science: Alberts, et al.’s Molecular Biology of
    the Cell, 4th edition: “Analyzing Protein Structure and Function”**
    Link:  National Center for Biotechnology Information’s Bookshelf:
    Garland Science: Alberts, et al.’s *Molecular Biology of the Cell,
    4<sup>th</sup> edition:  *“[Analyzing Protein Structure and
    Function](http://www.ncbi.nlm.nih.gov/books/NBK26820/#A1606)”
    (HTML)  
        
     Instructions: Please study "Affinity Chromatography and
    Immunoprecipitation Allow Identification of Associated Proteins"
    section on this page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.4.3 Protein Arrays** <span id="4.4.3"></span> 
-   **Reading: Functional Genomics: Mike Taussig and Oda Stoevesandt’s
    “Protein Arrays Resource Page”**
    Link: Functional Genomics:  Mike Taussig and Oda Stoevesandt’s
    “[Protein Arrays Resource
    Page](http://www.functionalgenomics.org.uk/sections/resources/protein_arrays.htm)”
    (HTML)  
        
     Instructions: Please study this page.  Please note that there are
    different types of protein arrays.  Protein arrays represent a more
    recent development of high throughput screening compared to DNA
    microarrays.   
        
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.5 Recombinant Proteins** <span id="4.5"></span> 
**4.5.1 Expression by Bacteria** <span id="4.5.1"></span> 
-   **Reading: European Molecular Biology Laboratory’s “Protein
    Expression and Purification Core Facility: Protein Expression E.
    coli”**
    Link: European Molecular Biology Laboratory’s "[Protein Expression
    and Purification Core Facility: Protein Expression E.
    coli](http://www.embl.de/pepcore/pepcore_services/protein_expression/ecoli/index.html)”
    (HTML)  
        
     Instructions: Please study this page.  Please note that *E. coli*
    can be employed to express a functional protein only if the
    biological function of the protein does not require
    posttranslational modification.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.5.2 Expression by Yeast** <span id="4.5.2"></span> 
-   **Reading: John W. Kimball’s Biology Pages: “Gene Regulation in
    Eukaryotes”**
    Link: John W. Kimball’s Biology Pages: “[Gene Regulation in
    Eukaryotes](http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/P/Promoter.html#Gene_Regulation_in_Eukaryotes)”
    (HTML)  
        
     Instruction: Please study this page.  Focus on the differences
    between eukaryotic and prokaryotic genes expression, including
    posttranslational modifications.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: BioPharm International: Rudolph et al.’s “Expression of
    Recombinant Proteins in Yeast”**
    Links: BioPharm International: Rudolph et al.’s “[Expression of
    Recombinant Proteins in
    Yeast](http://biopharminternational.findpharma.com/biopharm/Article/Expression-of-Recombinant-Proteins-in-Yeast/ArticleStandard/Article/detail/301979)”
    (HTML)  
        
     Instructions: Please study this page.  Please note that *Pichia
    pastoris* and other yeast hosts have the capability to modify
    recombinant proteins posttranslationally.  Posttranslational
    modification is an essential feature of biologically active
    proteins.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.5.3 Expression by Mammalian Cells** <span id="4.5.3"></span> 
-   **Reading: BioMed Central: BMC Biotechnology: Nehlsen et al.’s
    “Recombinant Protein Expression by Targeting Pre-Selected
    Chromosomal Loci”**
    Link: BioMed Central: *BMC Biotechnology:  *Nehlsen et al.’s
    “[Recombinant Protein Expression by Targeting Pre-Selected
    Chromosomal Loci](http://www.biomedcentral.com/1472-6750/9/100)”
     (HTML or PDF)  
        
     Instructions: Please read the “Background” section found under
    “Abstract” on this page.  You can access the PDF version from the
    righthand side of the page, under "viewing options."  This is a
    peer-reviewed publication.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.5.4 Protein Glycosylation** <span id="4.5.4"></span> 
-   **Reading: National Center for Biotechnology Information's
    Bookshelf: Lodish, Berk, Zipursky, et al.’s Molecular Cell Biology,
    4th edition: “Protein Glycosylation in the ER and in Golgi
    Complex”**
    Link: National Center for Biotechnology Information's Bookshelf: 
    Lodish, Berk, Zipursky, et al.’s *Molecular Cell Biology,
    4<sup>th</sup> edition:*  “[Protein Glycosylation in the ER and in
    Golgi Complex](http://www.ncbi.nlm.nih.gov/books/NBK21744/)”
    (HTML)  
        
     Instructions: Please study this entire section from Lodish, Berk,
    Zipurksy, et al.’s *Molecular Cell Biology* textbook.  Please note
    that protein glycosylation is very important for certain proteins to
    function properly in eukaryotes.  Proteins that are expressed in
    prokaryotic cell, for example, in E. coli, cannot be glycosylated,
    because these cells lack membrane-bound subcellular organelles.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.6 Protein Engineering** <span id="4.6"></span> 
**4.6.1 Protein Folding in Biotechnology** <span id="4.6.1"></span> 
-   **Reading: Scientific Electronic Library Online Brazil: Brazilian
    Journal of Medical and Biological Research: J.M. Yon’s “Protein
    Folding: A Perspective for Biology, Medicine and Biotechnology”**
    Link: Scientific Electronic Library Online Brazil: *Brazilian
    Journal of Medical and Biological Research:* J.M. Yon’s “[Protein
    Folding: A Perspective for Biology, Medicine and
    Biotechnology](http://www.scielo.br/scielo.php?script=sci_arttext&pid=S0100-879X2001000400001&lng=en&nrm=iso&tlng=en)”
    (HTML or PDF)  
        
     Instructions: Please study the content of “Protein Folding in
    Biotechnology: Protein Engineering and Design” on this page.  You
    can access the PDF version from the right hand side of the page.
     Author Yon works at the Institut de Biochimie, Biophysique
    Moléculaire et Cellulaire, UMR CNRS, Université de Paris-Sud, Orsay,
    France.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.6.2 Enzyme Binding Site Specificity** <span id="4.6.2"></span> 
-   **Reading: Oxford University Press’s Journal of Experimental
    Biology: Jeanneau, et al.’s “Manipulating PEPC Levels in Plants”**
    Links: Oxford University Press’s *Journal of Experimental
    Biology: *Jeanneau, et al.’s “[Manipulating PEPC Levels in
    Plants](http://jxb.oxfordjournals.org/content/53/376/1837.long)”
    (HTML or PDF)  
        
     Instructions: Please study this article.  Review what you have
    learned about RiBisCo, and focus on the “Transgenic Plants” section
    on this page.  You can access the PDF version by clicking the “Full
    Text (PDF)” button on the right-hand side of the page.  This is a
    peer-reviewed publication.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.6.3 Structural Scaffolds and Domain Recombination** <span
id="4.6.3"></span> 
-   **Reading: Oxford University Press’s Nucleic Acids Research: Sleight
    et al.’s “In-Fusion BioBrick Assembly and Re-Engineering”**
    Links: Oxford University Press’s *Nucleic Acids Research:  *Sleight
    et al.’s “[In-Fusion BioBrick Assembly and
    Re-Engineering](http://nar.oxfordjournals.org/content/38/8/2624.long)”
    (HTML or PDF)  
        
     Instructions: Please study this entire article. Focus on the
    “Introduction,” “Results,” and “Discussion” sections.  You can
    access the PDF from the right-hand side of the page.  This is a
    peer-reviewed publication. The authors work at the Department of
    Bioengineering, University of Washington, Seattle.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.6.4 Nonnatural Amino Acids** <span id="4.6.4"></span> 
-   **Reading: The FASEB Journal: Budisa, et al.’s “Toward the
    Experimental Codon Reassignment In Vivo: Protein Building With an
    Expanded Amino Acid Repertoire”**
    Link: *The FASEB Journal:  *Budisa, et al.’s “[Toward the
    Experimental Codon Reassignment In Vivo: Protein Building With an
    Expanded Amino Acid
    Repertoire](http://www.fasebj.org/content/13/1/41.long)” (HTML or
    PDF)  
        
     Instructions: Please focus on the " 'Restricted' vs. 'Relaxed'
    Genetic Code" and "Codon Reassignment In Living Cell" sections.  You
    can access the PDF from the right-hand side of the page.  The
    authors work at the Max Planck Institut für Biochemie, Germany.
     This is a peer-reviewed publication.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.6.5 Domain Recombination** <span id="4.6.5"></span> 
-   **Reading: BioMed Central: BMC Medicine: David Davis and David
    Stokoe’s “Zinc Finger Nucleases as Tools To Understand and Treat
    Human Diseases”**
    Link: BioMed Central:  *BMC Medicine*:  David Davis and David
    Stokoe’s “[Zinc Finger Nucleases as Tools To Understand and Treat
    Human Diseases](http://www.biomedcentral.com/1741-7015/8/42)” (HTML
    or PDF)  
        
     Instructions: Please focus on the following sections: “Methods for
    Design, Testing and Implementation of Zinc Finger Proteins (ZFPs):”
    “Addition of Functional Domains Expands the Utility of ZFPs:” and
    “Addition of Eendonuclease Activities to ZFPs to Create Targeted DNA
    Scissors.”  Please note that domain recombination is the recombining
    of two known functional domains of proteins to create a new kind of
    protein.  You can access the PDF from the right-hand side of the
    page.  Authors work in the Department of Molecular Biology at
    Genentech Inc.  This is a peer-reviewed publication.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.6.6 DNA Shuffling** <span id="4.6.6"></span> 
-   **Reading: National Center for Biotechnology Information’s
    Bookshelf: Wiley-Liss: Strachan and Read’s Human Molecular Genetics,
    2nd edition: “Chapter 14: Our Place in the Tree of Life”**
    Link: National Center for Biotechnology Information’s Bookshelf:
     Wiley-Liss: Strachan and Read’s *Human Molecular Genetics,
    2<sup>nd</sup> edition: * “[Chapter 14: Our Place in the Tree of
    Life](http://www.ncbi.nlm.nih.gov/books/NBK7565/#A1749)” (HTML)  
        
     Instructions: Please study the “14.5.2 Exon Shuffling Permits
    Diverse Combinations of Structure and Functional Modules, and May be
    Mediated by Transposable Elements” section on this page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.6.7 Combinatory Protein Libraries** <span id="4.6.7"></span> 
-   **Reading: National Center for Biotechnology Information’s PubMed:
    PLoS Biology: David R. Halpin and Pehr B. Harbury’s “DNA Display II.
    Genetic Manipulation of Combinatorial Chemistry Libraries for
    Small-Molecule Evolution”**
    Link: National Center for Biotechnology Information’s PubMed:  *PLoS
    Biology:*  David R. Halpin and Pehr B. Harbury’s “[DNA Display II.
    Genetic Manipulation of Combinatorial Chemistry Libraries for
    Small-Molecule
    Evolution](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC434149/)”
    (HTML or PDF)  
        
     Instructions: Please study “Strategy” in the “Results” section. 
    You can access the PDF from the top right corner of the page.  The
    two authors, Halpin and Harbury, work in the Department of
    Biochemistry at Stanford University’s School of Medicine. This is a
    peer-reviewed publication.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**4.6.8 Biomaterials** <span id="4.6.8"></span> 
-   **Reading: National Center for Biotechnology Information’s
    Bookshelf: Harvard Stem Cell Institute: Stephanie M. Willerth and
    Shelly E. Sakiyama-Elbert’s StemBook: “Combining Stem Cells and
    Biomaterial Scaffolds for Constructing Tissues and Cell Delivery”**
    Link:  National Center for Biotechnology Information’s Bookshelf:
    Harvard Stem Cell Institute: Stephanie M. Willerth and Shelly E.
    Sakiyama-Elbert’s *StemBook:* “[Combining Stem Cells and Biomaterial
    Scaffolds for Constructing Tissues and Cell
    Delivery](http://www.ncbi.nlm.nih.gov/books/NBK27050/)” (HTML or
    PDF)  
        
     Instructions:  Please study this page.  You can access the PDF from
    the right-hand side of the page under the “Download” section. The
    two authors, Willerth and Sakiyama-Elbert, are affiliated with the
    Department of Biomedical Engineering at Washington University, St.
    Louis.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.


