**Unit 3: Genomics and Gene Expression Technology** <span
id="3"></span> 
*This unit looks at how genome projects are performed and how systemic
studies approach the complexity of organisms.  In 2003, biologists
reached a milestone in biotechnology when sequencing a human genome was
completed.  *  
 *             
 We will look at the types of information contained in the genome map. 
Next, we will examine high throughput comparative techniques, namely
microarrays and bioinformatics.  Lastly, we will look at the impact of
genomics in medicine and evolution.*

**Unit 3 Time Advisory**  
This unit should take you approximately 24 hours to complete.  
  
 ☐    Subunit 3.1: 1.0 hour  
  
 ☐    Subunit 3.2: 2.5 hours  
  
 ☐    Subunit 3.3: 2.0 hours  
  
 ☐    Subunit 3.4: 5.0 hours  
  
☐    Subunit 3.4.1: 1.0 hour  
  
 ☐    Subunit 3.4.2: 1.0 hour  
  
 ☐    Subunit 3.4.3: 0.5 hour  
  
 ☐    Subunit 3.4.4: 0.5 hour  
  
 ☐    Subunit 3.4.5: 1.0 hour  
  
 ☐    Subunit 3.4.6: 1.0 hour

☐    Subunit 3.5: 3.0hours  
  
 ☐    Subunit 3.6: 2.5 hours  
  
 ☐    Subunit 3.7: 2.0 hours  
  
 ☐    Subunit 3.8: 2.0 hours  
  
 ☐    Subunit 3.9: 4.0 hours  
  
☐    Subunit 3.9.1: 1.0 hour  
  
 ☐    Subunit 3.9.2: 1.0 hour  
  
 ☐    Subunit 3.9.3: 1.0 hour  
  
 ☐    Subunit 3.9.4: 1.0 hour

**Unit3 Learning Outcomes**  
Upon successful completion of this unit, the student will be able to:  
  
-   Compare genetic and physical genome maps.
-   Describe the design of genome projects.
-   Compare and contrast systemic studies.
-   Describe the significance of genetic makeup in personalized medical
    treatment.
-   Discuss the link between gene families and genetic evolution

**3.1 Genome Maps** <span id="3.1"></span> 
**3.1.1 Genetic Maps** <span id="3.1.1"></span> 
-   **Reading: Nature Education’s Scitable: Ingrid Lobo and Kenna Shaw’s
    “Thomas Hunt Morgan, Genetic Recombination, and Gene Mapping”**
    Link: Nature Education’s Scitable:  Ingrid Lobo and Kenna Shaw’s
    “[Thomas Hunt Morgan, Genetic Recombination, and Gene
    Mapping](http://www.nature.com/scitable/topicpage/thomas-hunt-morgan-genetic-recombination-and-gene-496)”
    (HTML)  
        
     Instructions: Please recall crossing over events during meiosis
    while you are studying this publication.  Crossing over causes
    genetic recombination, and it contributes to offspring diversity.
     Crossing over events can be utilized to construct genetic linkage
    maps. Genetic linkage maps place genes at a certain distance on a
    chromosome.  The placement of the genes is relative to each other;
    their absolute distance stays unknown.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.1.2 Physical Maps** <span id="3.1.2"></span> 
-   **Reading: John W. Kimball’s Biology Pages: “ Genetic Linkage and
    Genetic Maps”**
    Link: John W. Kimball’s Biology Pages: “[Genetic Linkage and Genetic
    Maps](http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/L/Linkage.html)”
    (HTML)  
        
     Instructions: Please study the “Genetic versus Physical Maps”
    section on the page.  Please note that physical maps provide the
    absolute distance of the genes on the chromosome.  In situ
    hybridization of genes on a chromosome is a technique that results
    in a physical map.  The best physical maps derive from genome
    sequencing projects.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.2 Genetic Markers for Mapping** <span id="3.2"></span> 
**3.2.1 RFLPs (Restriction Fragment Length Polymorphisms)** <span
id="3.2.1"></span> 
-   **Reading: National Center for Biotechnology Information’s
    “Restriction Fragment Length Polymorphism (RFLP)”**
    Link:  National Center for Biotechnology
    Information’s  “[Restriction Fragment Length Polymorphism
    (RFLP)](http://www.ncbi.nlm.nih.gov/projects/genome/probe/doc/TechRFLP.shtml)”
    (HTML)  
        
     Instructions: Please study this page.  Restriction fragment length
    polymorphism has been instrumental in mapping genes and in medical
    diagnosis.  RFLP was the first DNA technique employed in forensic
    laboratories for the identification of individuals.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.2.2 SNPs (Single Nucleotide Polymorphisms)** <span
id="3.2.2"></span> 
-   **Reading: National Center for Biotechnology Information’s “SNPs:
    Variations on a Theme”**
    Link:  National Center for Biotechnology Information’s “[SNPs:
    Variations on a
    Theme](http://www.ncbi.nlm.nih.gov/About/primer/snps.html)” (HTML)  
        
     Instructions: Please study this page.  Single nucleotide
    polymorphism results from point mutations in a certain position of
    the DNA sequence.  Point mutations may or may not have biological
    consequences, depending on whether they change the function of the
    gene product.  SNPs have been used in medical diagnosis.  A
    combination of SNPs can be used for identifying individuals and
    species.    
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.2.3 VNTRs (Variable Number Tandem Repeats)** <span
id="3.2.3"></span> 
-   **Reading: Nature Education’s Scitable: P. Z. Myers’ “Tandem Repeats
    and Morphological Variation”**
    Link:   Nature Education’s Scitable:  P. Z. Myers’ “[Tandem Repeats
    and Morphological
    Variation](http://www.nature.com/scitable/topicpage/tandem-repeats-and-morphological-variation-40690)”
    (HTML)  
        
     Instructions: Please study this page to learn about how repeat
    number variations can identify individuals in a population.  Robust
    change in the number of repeats in certain genetic positions is also
    linked to phenotypic changes.  Author Dr. Myers works at the
    University of Minnesota.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.2.4 Microsatellite Polymorphisms** <span id="3.2.4"></span> 
-   **Reading: Davidson College’s: Department of Biology’s
    “Microsatellite DNA Methodology”**
    Link: Davidson College’s: Department of Biology’s  “[Microsatellite
    DNA
    Methodology](http://www.bio.davidson.edu/courses/genomics/method/microsatellite.html)”
    (HTML)  
        
     Instructions: Please study this page.  Note that microsatellite
    polymorphism is similar to VNTRs, but the size of the repeats is
    shorter.  Many applications refer to it as short tandem repeats
    (STRs).  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.3 Physical Sequence Data** <span id="3.3"></span> 
**3.3.1 Sequence Tagged Sites (STSs)** <span id="3.3.1"></span> 
-   **Reading: National Human Genome Research Institute’s
    “Sequence-Tagged Sites, Another Marker”**
    Link:  National Human Genome Research Institute’s  “[Sequence-Tagged
    Sites, Another Marker](http://www.genome.gov/25520328)” (HTML)  
        
     Instructions: Please study this page.  Please note that sequence
    tagged sites are fundamental in recent genome project techniques.
     STS sequences have no particular biological significance.  STS is a
    term used when we refer to a method.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.3.2 ESTs (Expressed Sequence Tags)** <span id="3.3.2"></span> 
-   **Reading: National Center for Biotechnology Information’s “ESTs:
    Gene Discovery Made Easier”**
    Link: National Center for Biotechnology Information’s “[ESTs: Gene
    Discovery Made
    Easier](http://www.ncbi.nlm.nih.gov/About/primer/est.html)” (HTML)  
        
     Instructions: Please study this page. ESTs serve us well in genome
    projects and gene expression studies.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.3.3 Contigs** <span id="3.3.3"></span> 
-   **Reading: JGI Genome Portal’s “What Is a Scaffold?”**
    Link:  JGI Genome Portal’s  “[What Is a
    Scaffold?](http://genome.jgi-psf.org/help/scaffolds.html)” (HTML)  
        
     Instructions: Please study this page.  Please note that a contig is
    a set of aligned, overlapping, nucleic acid sequences.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.3.4 Radiation Hybrid Mapping** <span id="3.3.4"></span> 
-   **Reading: Biology Reference’s: Christine Klein’s “Radiation Hybrid
    Mapping”**
    Link: Biology Reference’s: Christine Klein’s  “[Radiation Hybrid
    Mapping](http://www.biologyreference.com/Po-Re/Radiation-Hybrid-Mapping.html)”
    (HTML)  
        
     Instructions: Please study this page.  Please note the similarities
    to mapping with genetic recombination (BIO403 Subunit 3.1.1 Genetic
    Maps).  Radiation mapping is used to determine the relative distance
    between chromosomal markers.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.3.5 Cytogenetic Mapping** <span id="3.3.5"></span> 
-   **Reading: Blackwell Publishing Ltd: Animal Genetics: G. P. Di Meo
    et al.’s “An Advanced Sheep (Ovis aries, 2n = 54) Cytogenetic Map
    and Assignment of 88 New Autosomal Loci by Fluorescence In Situ
    Hybridization and R-Banding”**
    Link: Blackwell Publishing Ltd:*Animal Genetics:  *G. P. Di Meo et
    al.’s “[An Advanced Sheep (Ovis aries, 2n = 54) Cytogenetic Map and
    Assignment of 88 New Autosomal Loci by Fluorescence *In Situ*
    Hybridization and
    R-Banding](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2063634/)”
    (HTML or PDF)  
        
     Instructions: Please study the “Introduction” section on this
    page.  Cytogenetic mapping generates a physical map of genes on a
    chromosome.  You can access the PDF form from the top right corner
    of the page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.4 Human Genome and Human Genome Project** <span id="3.4"></span> 
**3.4.1 Chromosome Walking** <span id="3.4.1"></span> 
-   **Reading: Davidson College’s: Department of Biology’s “Chromosomal
    Walking to Clone the Cystic Fibrosis Gene”**
    Link: Davidson College’s: Department of Biology’s  “[Chromosomal
    Walking to Clone the Cystic Fibrosis
    Gene](http://www.bio.davidson.edu/courses/genomics/method/Chromwalk.html)”
    (HTML)  
        
     Instructions: Please study this page.  Chromosomal walking can be
    used to determine the position of a gene in the genome.  It is
    employed only if the genome of the species is not sequenced.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.4.2 Shotgun Sequencing** <span id="3.4.2"></span> 
-   **Reading: Genome News Network: Bijal P. Trivedi’s “Sequencing the
    Genome”**
    Link:  Genome News Network: Bijal P. Trivedi’s  “[Sequencing the
    Genome](http://www.genomenewsnetwork.org/articles/06_00/sequence_primer.shtml)”
    (HTML)  
        
     Instructions: Please study this page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.  The shotgun genome sequencing
    method has been designed by Craig Venter.  This method skips
    physical mapping.

**3.4.3 Heterochromatin Gaps** <span id="3.4.3"></span> 
-   **Reading: Nature Education’s Scitable: “Chromosomes”**
    Link: Nature Education’s
    Scitable: “[Chromosomes](http://www.nature.com/scitable/topicpage/chromosomes-14121320)”
    (HTML)  
        
     Instructions: Please study the “Why Is Complex Packing Critical for
    Eukaryotic Chromosomes?” section on this page.  Please note that
    heterochromatins are tightly packed regions of the genome at the
    centromer and telomer regions.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: National Human Genome Project Institute’s “Drosophila
    Heterochromatin Genome Project”**
    Link:  National Human Genome Project Institute’s  “[Drosophila
    Heterochromatin Genome
    Project](https://web.archive.org/web/20120624195208/http://www.dhgp.org/index_about.html)”
    (HTML)  
        
     Instructions: Please read the “Background” section on this page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.4.4 Relative Components of Human Genome** <span id="3.4.4"></span> 
-   **Reading: U.S. Department of Energy, Office of Science’s Human
    Genome Project Information: “About the Human Genome Project”**
    Link:  U.S. Department of Energy, Office of Science’s Human Genome
    Project Information:  “[About the Human Genome
    Project](http://www.ornl.gov/sci/techresources/Human_Genome/project/about.shtml)”
    (HTML)  
        
     Instructions: Please study this page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.  Human genome sequencing has been
    greatly speeded up by the shotgun sequencing method.  The sequencing
    of genomic regions that are rich in repetitive sequences, such as
    telomeres and centromeres, are still in progress.

**3.4.5 Estimated Gene Content** <span id="3.4.5"></span> 
-   **Reading: U.S. Department of Energy, Office of Science’s Human
    Genome Project Information: “How Many Genes are in the Human
    Genome?”**
    Link:  U.S. Department of Energy, Office of Science’s Human Genome
    Project Information:  “[How Many Genes are in the Human
    Genome?](http://www.ornl.gov/sci/techresources/Human_Genome/faq/genenumber.shtml)”
    (HTML)  
        
     Instructions: Please study this page.  Please note that we have
    only estimations on the number of genes in the human genome.  These
    predictions are aided by bioinformatics but should be verified in
    the future by experiments.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.4.6 Pseudogenes** <span id="3.4.6"></span> 
-   **Reading: Yale University’s Gerstein Lab: “Pseudogene.org: Genome
    Analysis”**
    Link:  Yale University’s Gerstein Lab:
    “[Pseudogene.org](http://pseudogene.org/background.php) Genome
    Analysis” (HTML)  
        
     Instructions: Please study this page.  Follow the link in "What
    causes pseudogenes to arise?" box for a graphical illustration.
     Please note that pseudogenes do not have a known biological
    function.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.4.7 Noncoding DNA Types, Including Junk DNA** <span
id="3.4.7"></span> 
-   **Reading: National Center for Biotechnology Information’s
    Bookshelf: W.H. Freeman: Lodish, et al.’s Molecular Cell Biology,
    4th edition: “Chromosomal Organization of Genes and Noncoding DNA”**
    Link:  National Center for Biotechnology Information’s Bookshelf:
     W.H. Freeman: Lodish, et al.’s *Molecular Cell Biology,
    4<sup>th</sup> edition:  *“[Chromosomal Organization of Genes and
    Noncoding DNA](http://www.ncbi.nlm.nih.gov/books/NBK21571/)”
    (HTML)  
        
     Instructions: Please study the “Genomes of Higher Eukaryotes
    Contain Much Nonfunctional DNA” section on this page.  Please note
    that some DNA regions are genes coding proteins or functional RNA
    products, while other regions are noncoding.  Some of the noncoding
    regions, for example, repeat sequences and pseudogenes and have no
    known function.  These seemingly useless regions are referred as
    “junk DNA.”  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.5 DNA Microarrays** <span id="3.5"></span> 
**3.5.1 Monitoring Gene Expression** <span id="3.5.1"></span> 
-   **Reading: National Human Genome Research Institute’s “DNA
    Microarray Technology”**
    Link:  National Human Genome Research Institute’s  “[DNA Microarray
    Technology](http://www.genome.gov/10000533)” (HTML)  
        
     Instructions: Please study this page.  Please note that microarray
    is an efficient and widely used high throughput technology of
    differential gene expression studies.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Lecture: YouTube: Proneural’s “DNA Chips and Microarrays”**
    Link: YouTube: Proneural’s “[DNA Chips and
    Microarrays](http://www.youtube.com/watch?v=ui4BOtwJEXs)”
    (YouTube)  
        
     Instructions: Please watch the brief, 2-minute video.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.5.2 Comparative Studies** <span id="3.5.2"></span> 
-   **Reading: Nature Education’s Scitable: “Scientists Can Study an
    Organism’s Entire Genome with Microarray Analysis”**
    Link: Nature Education’s Scitable:  “[Scientists Can Study an
    Organism’s Entire Genome with Microarray
    Analysis](http://www.nature.com/scitable/topicpage/scientists-can-study-an-organism-s-entire-6526266)”
    (HTML)  
        
     Instructions: Please study this page and watch the video.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Web Media: Davidson College: Department of Biology’s “DNA
    Microarray Methodology”**
    Link: Davidson College: Department of Biology’s “[DNA Microarray
    Methodology](http://www.bio.davidson.edu/courses/genomics/chip/chip.html)”
    (Adobe Flash)  
        
     Instructions: Please watch the animation.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.5.3 ChIP-Chip (Chromatin Immunoprecipitation)** <span
id="3.5.3"></span> 
-   **Reading: Brandeis University’s: Jim Haber’s “Chromatin
    Immunoprecipitation”**
    Link: Brandeis University’s: Jim Haber’s  “[Chromatin
    Immunoprecipitation](http://www.bio.brandeis.edu/haberlab/jehsite/chIP.html)”
    (HTML)  
        
     Instructions: Please study this page, and follow the “protocol”
    link on the bottom of the page.  Chromatin immunoprecipitation is
    used to identify interacting proteins and DNA segments.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.6 Bioinformatics and Genomics** <span id="3.6"></span> 
**3.6.1 Data Mining** <span id="3.6.1"></span> 
-   **Reading: Bio Databases: Houle, et al.’s “Database Mining in the
    Human Genome Initiative”**
    Link: Bio Databases: Houle, et al.’s “[Database Mining in the Human
    Genome
    Initiative](http://www.biodatabases.com/whitepaper01.html#01)”
    (HTML)  
        
     Instructions: Please learn the content of the white paper, from the
    “Abstract” to the “Conclusion.”  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.6.2 Comparative Genomics** <span id="3.6.2"></span> 
-   **Reading: PLoS Biology: Ross C. Hardison’s “Comparative Genomics”**
    Link:  *PLoSBiology:  *Ross C. Hardison's  “[Comparative
    Genomics](http://www.plosbiology.org/article/info:doi/10.1371/journal.pbio.0000058)”
    (HTML or PDF)  
        
     Instructions: Please study this publication. You can access the PDF
    format from the top right corner of the page.  Genome sequence
    comparisons can be used to predict gene function between species and
    between the individuals of a species.  It is also used to determine
    phylogenetic distances among species.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: U.S. Department of Energy, Office of Science’s Human
    Genome Project Information: “Functional and Comparative Genomics
    Fact Sheet”**
    Link:  U.S. Department of Energy, Office of Science’s Human Genome
    Project Information:  “[Functional and Comparative Genomics Fact
    Sheet](http://www.ornl.gov/sci/techresources/Human_Genome/faq/compgen.shtml)”
    (HTML)  
        
     Instructions: Please study this page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.7 Medicine and Genomics** <span id="3.7"></span> 
**3.7.1 Gene Testing** <span id="3.7.1"></span> 
-   **Reading: U.S. Department of Energy, Office of Science’s Human
    Genome Project Information: “Gene Testing”**
    Link:  U.S. Department of Energy, Office of Science’s Human Genome
    Project Information:  “[Gene
    Testing](http://www.ornl.gov/sci/techresources/Human_Genome/medicine/genetest.shtml)”
    (HTML)  
        
     Instructions: Please study this page.  Gene testing is performed
    primarily in medical diagnosis. It is also a mostly historical
    forensic technique for the identification of individuals.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: National Health Museum’s “Understanding Gene Testing—What
    Does a Predictive Gene Test Tell You?”**
    Link:  National Health Museum’s  “[Understanding Gene Testing—What
    Does a Predictive Gene Test Tell
    You?](http://www.accessexcellence.org/AE/AEPC/NIH/gene14.php)”
    (HTML)  
        
     Instructions: Please study this page.  Please note that gene
    testing can link the presence of a gene to the probability of
    developing a condition (disease) in an individual or in the
    offspring of an individual.    
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: Nature Education’s Scitable: Leslie A. Pray’s
    “Questionable Prognostic Value of Genetic Testing”**
    Link:  Nature Education’s Scitable:  Leslie A. Pray’s “[Questionable
    Prognostic Value of Genetic
    Testing](http://www.nature.com/scitable/topicpage/questionable-prognostic-value-of-genetic-testing-784)”
    (HTML)  
        
     Instructions: Please study this page.  There is an increasing
    interest on the environmental factors on trait penetrance.  Research
    should unravel how to take advantage of a genetic prognosis.
     Environmental factors such as nutrition and personal habits, seem
    to influence gene expression and the penetrance of traits.
     Medications can also be developed to decrease the risk of undesired
    traits (disease).  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.7.2 Pharmacogenetics and Pharmacogenomics** <span
id="3.7.2"></span> 
-   **Reading: The Royal Society’s “What Is Pharmacogenetics?”**
    Link: The Royal Society’s “[What Is
    Pharmacogenetics?](http://royalsociety.org/News.aspx?id=6059)”
    (HTML)  
        
     Instructions: Please study this page.  Please note that individual
    genetic makeup will influence the response to a medication.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: Reading: U.S. Department of Energy, Office of Science’s
    Human Genome Project Information: “Pharmacogenomics”**
    Link:  U.S. Department of Energy, Office of Science’s Human Genome
    Project
    Information:  “[Pharmacogenomics](http://www.ornl.gov/sci/techresources/Human_Genome/medicine/pharma.shtml)”
    (HTML)  
        
     Instructions: Please study this page.  Pharmacogenomics is a way
    drugs can be designed in the future.  It requires knowing
    individuals' genetic makeup besides the risk factors what doctors
    use today.  It holds the promise of safer and more efficient
    medication.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.8 Genetic Evolution** <span id="3.8"></span> 
**3.8.1 Molecular Phylogenetics** <span id="3.8.1"></span> 
-   **Reading: National Center for Biotechnology Information’s
    “Systematics and Molecular Phylogenetics”**
    Link:  National Center for Biotechnology
    Information’s  “[Systematics and Molecular
    Phylogenetics](http://www.ncbi.nlm.nih.gov/About/primer/phylo.html)”
    (HTML)  
        
     Instructions: Please study this page.  Molecular phylogenetics
    establishes relationship between species based on their genetic
    material.  Before the availability of molecular techniques, species
    were classified based on their phenotype and behavior.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.8.2 Gene Superfamilies** <span id="3.8.2"></span> 
-   **Reading: Sandwalk: Larry A. Moran’s “The Evolution of Gene
    Families”**
    Link:  Sandwalk:  Larry A. Moran’s “[The Evolution of Gene
    Families](http://sandwalk.blogspot.com/2007/01/evolution-of-gene-families.html)”
    (HTML)  
        
     Instructions: Please study this page by Dr. Moran, Professor of
    Biochemistry at the University of Toronto.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.9 Other Systemic Studies** <span id="3.9"></span> 
**3.9.1 Transcriptome** <span id="3.9.1"></span> 
-   **Reading: Nature Education’s Scitable: “Systems Biology Allows Us
    to Think Broadly”**
    Link:  Nature Education’s Scitable: “[Systems Biology Allows Us to
    Think
    Broadly](http://www.nature.com/scitable/content/systems-biology-allows-us-to-think-broadly-14623200)”
    (HTML)  
        
     Instructions: Please study the diagram.  You may want to return to
    this diagram, as you are completing the remaining subunits in Unit
    3.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: Nature Education’s Scitable: Jill U. Adams’
    “Transcriptome: Connecting the Genome to Gene Function”**
    Link:  Nature Education’s Scitable: Jill U. Adams’  “[Transcriptome:
    Connecting the Genome to Gene
    Function](http://www.nature.com/scitable/topicpage/transcriptome-connecting-the-genome-to-gene-function-605)”
    (HTML)  
        
     Instructions: Please study this page by freelancer science writer
    Dr. Adams in its entirety. Please recall that the genetic material
    is identical in all somatic cells of a multicellular organism, but a
    different set of genes is expressed in different cells. The
    transcriptome is the collection of the expressed genes, and it is
    different in different cell types.   
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.9.2 Proteome** <span id="3.9.2"></span> 
-   **Reading: Nature Education’s Scitable: Jill Adams’ “The Proteome:
    Discovering the Structure and Function of Proteins”**
    Link: Nature Education’s Scitable: Jill Adams’ “[The Proteome:
    Discovering the Structure and Function of
    Proteins](http://www.nature.com/scitable/topicpage/the-proteome-discovering-the-structure-and-function-613)”
    (HTML)  
        
     Instructions: Please study this page by freelancer science writer
    Dr. Adams in its entirety. The proteome is the collection of
    proteins that are expressed.  Please note that the proteomes of
    different cell types are different.  Both proteome and transcriptome
    reflect gene activity.     
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.9.3 Metabolome** <span id="3.9.3"></span> 
-   **Reading: BioTechnique: Ute Roessner and Jairus Bowne’s “What Is
    Metabolomics All About?”**
    Link: BioTechnique: Ute Roessner and Jairus Bowne’s “[What Is
    Metabolomics All
    About?](http://www.biotechniques.com/BiotechniquesJournal/specialissues/2009/April/What-is-metabolomics-all-about/biotechniques-140692.html)”
    (HTML or PDF)  
        
     Instructions: Please study all three pages of this publication. 
    You can switch to the next page by clicking a number on the bottom
    of the page.  To access the PDF format, click the Full Text (PDF)
    button on the linked page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: Genome Alberta and Genome Canada’s “The Human Metabolome
    Project”**
    Link: Genome Alberta and Genome Canada’s  “[The Human Metabolome
    Project](http://www.metabolomics.ca/)” (HTML)  
        
     Instructions: Please study this page.  Please note that the
    metabolome consists of a variety of unrelated small molecular mass
    substances.  We usually know that these substances are present if
    they are above the detection limit of an employed analytical
    technique.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.9.4 Metagenome** <span id="3.9.4"></span> 
-   **Reading: Nature Education’s Scitable: Lorenz and Eck’s
    “Metagenomics and Industrial Applications”**
    Link: Nature Education’s Scitable:  Lorenz and Eck’s “[Metagenomics
    and Industrial
    Applications](http://www.nature.com/scitable/content/metagenomics-and-industrial-applications-56052)”
    (PDF)  
        
     Instructions: Please study this publication carefully.  Metagenome
    is the genetic information, which is extracted from an environmental
    microbial population.  It contains genetic material from multiple
    living organisms.    
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: Nature Education’s Scitable: Kira Zhaurova’s “Genomes of
    Other Organisms: DNA Barcoding and Metagenomics”**
    Link: Nature Education’s Scitable: Kira Zhaurova’s “[Genomes of
    Other Organisms: DNA Barcoding and
    Metagenomics](http://www.nature.com/scitable/topicpage/genomes-of-other-organisms-dna-barcoding-and-662)”
    (HTML)  
        
     Instructions: Please study this page.  Please compare the nature of
    full genome sequences and metagenomic data.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.


