---
layout: default
title: "BIO403: Biotechnology"
course_description: "A detailed introduction to the use of numerous methods and biological techniques such as FISH, RNAi, PCR, Western blotting, microarrays, and others to enhance scientific research, medicine, and engineering. Particular emphasis on how biotechnology and its techniques can be used in plant and animal engineering, gene therapy, forensic science, environmental studies, and renewable energy."
next: ../Unit03
previous: ../Unit01
---
**Unit 2: Advanced Nucleic Acid Technologies** <span id="2"></span> 
*This unit will discuss advanced nucleic acid technologies.  For
example, methods to visualize DNA and cloning vectors are used in
everything from creating gene therapies for disease to cleaning toxic
oil spills.  While DNA techniques have been established much longer than
RNA methods, RNA methods are exciting new and upcoming fields,
especially in ribozymes and RNA interference.  Both have far-reaching
implications in biotechnology that could lead to cures for disease and
entire new biomedical industries.*

**Unit 2 Time Advisory**  
This unit should take you approximately 9.5 hours to complete.  
  
 ☐    Subunit 2.1: 1.0 hour  
  
 ☐    Subunit 2.2: 2.0 hours  
  
 ☐    Subunit 2.3: 1.0 hour  
  
 ☐    Subunit 2.4: 5.5 hours  
  
☐    Subunit 2.4.1: 0.5 hour  
  
 ☐    Subunit 2.4.2: 2.0 hours  
  
 ☐    Subunit 2.4.3: 1.5 hours  
  
 ☐    Subunit 2.4.4: 1.5 hours

**Unit2 Learning Outcomes**  
Upon successful completion of this unit, the student will be able to:  
  
-   Describe advanced nucleic acid technologies.
-   Compare and contrast nucleic acid amplification techniques and
    determine the expected outcome.
-   Compare and contrast microRNA and RNAi.
-   Describe RNA techniques.    

**2.1 Fluorescence in Situ Hybridization (FISH)** <span
id="2.1"></span> 
-   **Reading: Nature Education's Scitable: Clare O’Connor’s
    “Fluorescence In Situ Hybridization”**
    Link: Nature Education's Scitable: Clare O’Connor’s “[Fluorescence
    *In Situ*
    Hybridization](http://www.nature.com/scitable/topicpage/fluorescence-in-situ-hybridization-fish-327)”
    (HTML)   
        
     Instructions: Please study the “*In Situ* Hybridization Is Used to
    Localize DNA Sequences on Chromosomes,” “Fluorescent Probes Are
    Introduced,” and “Using FISH to Identify the Positions of Genes”
    sections on this page.  Author Clare O’Connor works in the
    Department of Biology at Boston College.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2 Nucleic Acid Amplification** <span id="2.2"></span> 
**2.2.1 Modifications of PCR** <span id="2.2.1"></span> 
-   **Reading: Dr. Kary Banks Mullis’s “Polymerase Chain Reaction”**
    Link: Dr. Kary Banks Mullis’s “[Polymerase Chain
    Reaction](http://www.karymullis.com/pcr.shtml)” (HTML)  
        
     Instructions: Please read about Dr. Kary Bank Mullis.  It is rare
    that technological inventions receive a Nobel Prize, but Mullis was
    awarded the Nobel Prize in Chemistry in 1993 for his invention of
    the polymerase chain reaction (PCR) method.  Please study the main
    column and click on the “Questions About PCR” (PDF) link on the
    right and study that section as well.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: University of South Carolina's School of Medicine:
    Margaret Hunt’s “Real Time PCR”**
    Link: University of South Carolina's School of Medicine: Margaret
    Hunt’s “[Real Time
    PCR](http://pathmicro.med.sc.edu/pcr/realtime-home.htm)” (HTML)  
        
     Instructions: Please study this page carefully.  You can learn
    about an advanced PCR application, which is essential in tissue
    specific gene expression applications.  Please note that the first
    step is to reverse translate RNA to DNA, because we can amplify only
    DNA in the test tube.  Because of the reverse transcriptase step,
    this method is also called RT-PCR.  Here, it gets little confusing
    since the method is in real time as well, because it follows the PCR
    product production in real time.  The best way to keep all of these
    aspects in mind is to remember this method as “real time RT-PCR.” 
    Please make sure that you click on the thumbprints to enlarge all
    the images embedded in the left maroon-colored panel of this site.
       
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.2 Subtractive Hybridization** <span id="2.2.2"></span> 
-   **Reading: Evrogen’s “Suppression Subtractive Hybridization (SSH)”**
    Link: Evrogen’s “[Suppression Subtractive Hybridization
    (SSH)](http://www.evrogen.com/technologies/SSH.shtml)” (HTML)  
        
     Instructions: Please learn the content of this page.  Please note
    that subtractive hybridization is a technique that compares the
    differences between two nucleic acid pools and exponentially
    amplifies the differences in these pools by PCR.  The nucleic acid
    pools are commonly mRNA pools, thus such experiments generate
    subtraction cDNA libraries.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Assessment: The Saylor Foundation: "BIO403 Unit 2.2 Assessments"**
    Link: The Saylor Foundation: "[BIO403 Unit 2.2
    Assessments](http://www.saylor.org/content/bio403/assessments/BIO403_Unit_2_2_assessment_main.html)"
    (HTML)  
        
     Instructions: You will find links to the following four assessments
    on this page: "PCR," "RT-PCR," "Subtractive Hybridization," and
    "Differential Screening."  These are multiple choice assessments
    with one correct answer.  Clicking on an answer will bring you to
    another page.  If your answer is correct, then it is acknowledged
    with a short explanation.  Please read the explanation carefully and
    proceed to the next assessment as prompted at the bottom of the
    page.  If you clicked on the wrong answer, then the click will bring
    you to a tutorial page.  Please study the tutorial page carefully.
     At the end of the tutorial page, you will be prompted to return to
    the assessment and complete it again. You can also move between
    multiple choice pages using the links at the top of the page. Please
    complete the four assessments in order as the assessments are listed
    in increasing complexity.

**2.3 Chemical Synthesis of Genes** <span id="2.3"></span> 
-   **Reading: Wiley Online Library: Markus Fuhrmann, et al.’s “A
    Synthetic Gene Coding For the Green Fluorescent Protein (GFP) Is a
    Versatile Reporter in Chlamydomonas reinhardtii”**
    Link: Wiley Online Library: Markus Fuhrmann, et al.’s “[A Synthetic
    Gene Coding For the Green Fluorescent Protein (GFP) Is a Versatile
    Reporter in *Chlamydomonas
    reinhardtii*](http://onlinelibrary.wiley.com/doi/10.1046/j.1365-313X.1999.00526.x/full)”
    (HTML or PDF)  
        
     Instructions:  You may read this as HTML or a PDF.  To access the
    PDF, go to “Article Tools” on the right side of the page and click
    on “Get PDF.”  This is a peer-reviewed publication.  Please study
    the last paragraph of the “Introduction,” the entire “Synthesis of a
    C. Reinhardtii-Adapted GFP” in the “Results” section, and the entire
    “Discussion” section on this page.  The aim of this project is to
    exchange the codons in the jellyfish GFP gene to codons that are
    more frequent in a unicellular alga.  Please note that the codon
    preference varies in different organisms.   
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Assessment: The Saylor Foundation: "BIO403 Unit 2.3 Assessments"**
    Link: The Saylor Foundation: "[BIO403 Unit 2.3
    Assessments](http://www.saylor.org/content/bio403/assessments/BIO403_Unit_2_3_assessment_Codon_Preference.html)"
    (HTML)  
      
     Instructions: You will find the following "Codon Preference"
    assessment here. This is a multiple choice assessment with one
    correct answer.  Clicking on an answer will bring you to another
    page.  If your answer is correct, then it is acknowledged with a
    short explanation.  Please read the explanation carefully.  If you
    clicked on a wrong answer, then the click will bring you to a
    tutorial page.  Please study the tutorial page carefully.  On the
    tutorial page, you will be prompted to return to the assessment and
    complete it again. You can also move back to the assessment main
    page using the link at the top of the page.

**2.4 RNA Methods** <span id="2.4"></span> 
**2.4.1 Regulatory RNAs** <span id="2.4.1"></span> 
-   **Reading: Nature Education’s Scitable: Suzanne Clancy’s “RNA
    Functions”**
    Link:  Nature Education’s Scitable:  Suzanne Clancy’s “[RNA
    Functions](http://www.nature.com/scitable/topicpage/rna-functions-352)”
    (HTML)  
        
     Instructions: Please study this summary on the regulatory RNAs. 
    Here you will learn that besides the familiar mRNA, tRNA, and rRNA,
    other RNA functions are abundant in the cell.  The following
    sections will introduce you to several RNA techniques that are
    designed to modulate gene expression in the cell.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.2 RNAi (RNA Interference)** <span id="2.4.2"></span> 
-   **Reading: Macalester College: Mary Montgomery’s “RNAi”**
    Link: Macalester College: Mary Montgomery’s
    “[RNAi](http://www.macalester.edu/~montgomery/rnai.html)” (HTML)  
        
     Instructions: Please study this page.  Please note that RNA
    interference is an experimental technique.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.3 Natural Ribozymes** <span id="2.4.3"></span> 
-   **Reading: John W. Kimball’s Biology Pages: “Ribozymes”**
    Link: John W. Kimball’s Biology Pages:
     “[Ribozymes](http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/R/Ribozymes.html)”
    (HTML)  
        
     Instructions: Please study this page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.4 Synthetic Ribozymes** <span id="2.4.4"></span> 
-   **Reading: National Center for Biotechnology Information’s PubMed:
    American Society for Clinical Investigation: Nassim Usman and
    Lawrence M. Blatt’s “Nuclease-Resistant Synthetic Ribozymes:
    Developing a New Class of Therapeutics”**
    Link: National Center for Biotechnology Information’s
    PubMed:American Society for Clinical Investigation:  Nassim Usman
    and Lawrence M. Blatt’s “[Nuclease-Resistant Synthetic Ribozymes:
    Developing a New Class of
    Therapeutics](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC381444/)”
    (HTML or PDF)  
        
     Instructions: Please study this publication on making synthesized
    ribozymes. This material is also available in PDF form from the top
    right corner of the page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.5 Riboswitches** <span id="2.4.5"></span> 
-   **Reading: National Center for Biotechnology Information’s PubMed:
    Journal ACS Chemical Biology: Shana Topp and Justin P. Gallivan’s
    “Emerging Applications of Riboswitches in Chemical Biology”**
    Link:  National Center for Biotechnology Information’s PubMed:
    *Journal ACS Chemical Biology*:  Shana Topp and Justin P. Gallivan’s
    “[Emerging Applications of Riboswitches in Chemical
    Biology](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2811087/)”
    (HTML or PDF)  
        
     Instructions: Please study this page carefully.  This publication
    is challenging, but it helps you appreciate the regulatory roles of
    RNA in the cell.  Please rely heavily on the illustrations. It is
    not essential to memorize the names of the metabolites.  This
    material can also be viewed in PDF form from the top right corner of
    the page.  The authors work at Emory University.  This is a
    peer-reviewed publication.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.


